charts/sourcegraph/examples/kustomize-chart/README.md

```diff
@@ -0,0 +1,17 @@
+# kustomize chart
+
+This example demonstrate the escape hatch whenever you encounter something that our helm charts do not support yet. [Kustomize] allows you to apply [strategic merge patch] and [json patch] on helm chart without maintaining a custom fork. This is our **recommended** approach to customize helm chart, and you should avoid forking unless absolutely necessary.
```

Comment:
Maybe add an extra "if you do this, file a feature request" sentence?

Reply:
We should definitely do it. However, we haven't figured out how customers can provide helm feedback yet.

I'll start a thread on slack later today or tomorrow